.github/workflows/test.yaml

```diff
@@ -65,7 +75,18 @@ jobs:
           channels: pyviz/label/dev,bokeh/label/dev,conda-forge,nodefaults
       - name: Fetch unshallow
         run: git fetch --prune --tags --unshallow
+      - name: Get Today's date for cache
+        run: echo "TODAY=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
+      - name: conda cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ matrix.path }}
+          key: ${{ runner.os }}-conda-${{ matrix.python-version }}-${{ hashFiles('setup.py') }}-${{ env.TODAY }}-${{ env.CACHE_VERSION }}
```

Comment:
It'd be nice to have the installed group extras added in the cache key, as done in hvplot:

https://github.com/holoviz/hvplot/blob/04f6dae12c449987abf69601be47c1e292e1ab3a/.github/workflows/test.yaml#L61

We may use caching for the docs workflow for instance (we sometimes trigger many docs build in a day, that could help), and need to make sure the tests key not the same as the docs key. Maybe instead of adding the extras group it'd make more sense to add a hash of the test.yaml file itself?